### PR TITLE
fix: open only PDFs in new tabs

### PR DESCRIPTION
**Pull request by tmerten**
This looks at the file extension before converting links to be opened in a tab. This way `.docx`s or `.tar.gz`s are still being downloaded. 

Tested on Firefox and Chromium.

Checking the actual mime type might be a bit overkill but could be another (more robust) solution.

### CODE REVIEW

**Comment by anthonydillon** [comment]
That's a rather large lock file change. Do you know what caused it?

File: written-interview-in-new-tab.user.js [hunk 1]
```diff
@@ -15,11 +15,17 @@
...
-];
+const downloadLinks = Array.from(document.querySelectorAll(
+  "a[data-download-url*='person_attachments/downloads']"
+)).filter(el => el.textContent.endsWith(".pdf"));
+
+//   [
+//     ...document.querySelectorAll(
+//         "a[data-download-url*='person_attachments/downloads']"
+//     ),
+// ];
```
Comment (anthonydillon) [inline written-interview-in-new-tab.user.js:26]: Can you remove the commented out code here, please?

**Review by anthonydillon — CHANGES_REQUESTED** [review]

**Comment by tmerten** [comment]
> That's a rather large lock file change. Do you know what caused it?

This was due to a fresh npm install. It should be withing the boundaries of the versions defined in `package.json`.

File: written-interview-in-new-tab.user.js (hunk 1 above)
Comment (tmerten) [reply written-interview-in-new-tab.user.js]: How did these sneak in? Should be gone with the latest commit.

**Comment by tmerten** [comment]
Hey @anthonydillon . Thanks for the comments. I updated the PR. I'm also happy to revert the `.lock` file; the updates are not needed, just thought it's some maintenance :).

**Comment by anthonydillon** [comment]
LGTM thanks 👍

**Comment by tmerten** [comment]
> @tmerten do you want to bump the version of the script in the head of the document so this change is rolled out?

ah. this is how it works. bumped to `1.0.1`

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**